src/assets/stylesheets/main/layout/_nav.scss

```diff
@@ -571,7 +571,7 @@
 
       // Rotate icon for expanded lists
       .md-nav__item--nested .md-nav__toggle:checked ~ .md-nav__link & {
-        transform: rotate(90deg);
+        transform: translate(0, 50%) rotate(90deg);
```

Comment:
Even though it's transforming to 50%, it doesn't seem to be centered yet, look:

Current behavior:

https://user-images.githubusercontent.com/38889364/103663523-f2d52e00-4f4f-11eb-8d52-716c69b77c4a.mp4

Expected:

https://user-images.githubusercontent.com/38889364/103663542-fa94d280-4f4f-11eb-9c7c-0acbc77a4afd.mp4

WDYT?

Reply:
Seems a nice to have to me. Maybe we should put in an issue. Wdyt?